error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/StreamRules.java

```diff
@@ -788,4 +788,17 @@ Stream<T> after(T e1, T e2, T e3, T e4, T e5) {
       return Stream.of(e1, e2, e3, e4, e5);
     }
   }
+
+  // This improves performance by avoiding a full sort before removing duplicates.
+  static final class StreamSortedThenDistinct<T extends Comparable<? super T>> {
```

Comment:
By convention, we name the class after the result of the rewrite process
```suggestion
  static final class StreamDistinctSort<T extends Comparable<? super T>> {
```